packages/fxa-auth-server/lib/senders/emails/rtl.scss

```diff
@@ -0,0 +1,17 @@
+.ltr {
```

Comment:
Since this also explicitly sets `ltr`, maybe we can call this `ltr-rtl.scss`, or `locale-dir.scss`?